pages/family.tsx

```diff
@@ -1,13 +1,50 @@
 import * as React from 'react'
+import styled from 'styled-components'
 import { AddFamily, StyledFamilyList } from '../src/features/Family'
 
+const FamilyPageWrapper = styled.section`
+   h1 {
```

Comment:
indent must be 2 spaces

Reply:
https://github.com/aretecode/konsilos/pull/11/commits/08bf0476a81a57ca46badc71f1e25d5574dc2fbc